# Результат запуску тестів python manage.py test board.board_tests:
..
(.venv) D:\Documents\Python\Hillel\HomeWorks\hw_16_models_in_django\my_board_project
python manage.py test board.board_tests
Found 4 test(s).
Creating test database for alias 'default'...
System check identified no issues (0 silenced).
....
----------------------------------------------------------------------
Ran 4 tests in 0.007s

OK
Destroying test database for alias 'default'...
..
